LayoutTests/imported/w3c/web-platform-tests/scroll-to-text-fragment/scroll-to-text-fragment-target.html

```diff
@@ -42,6 +42,7 @@
   let results = {
     scrollPosition: position,
     href: window.location.href,
+    performanceEntry: performance.getEntries()[0].name,
```

Comment:
Is this change done in the upstream?
We should mention that in the commit message.

Reply:
No, we need to upstream it when this is pushed, which I understand is the easier direction?